pyunlocbox/tests/test_all.py

```diff
@@ -8,11 +8,11 @@
 
 import unittest
 
-from . import test_functions
-from . import test_operators
-from . import test_solvers
-from . import test_acceleration
-from . import test_docstrings
+from pyunlocbox.tests import test_functions
+from pyunlocbox.tests import test_operators
+from pyunlocbox.tests import test_solvers
+from pyunlocbox.tests import test_acceleration
+from pyunlocbox.tests import test_docstrings
```

Comment:
Why changing the imports from relative to absolute? There might be a risk that it imports an installed version of the `pyunlocbox` rather than the one in the current directory. I'm not sure about best practices tough.

Reply:
Should be OK, we do it this way in the PyGSP.